tests/network/bindingplugin.go

```diff
@@ -172,5 +173,59 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(vmi.Status.Interfaces).To(HaveLen(1))
 			Expect(vmi.Status.Interfaces[0].Name).To(Equal(primaryIface.Name))
 		})
+
+		It("can establish communication between two VMs", func() {
+			const (
+				guestIfaceName = "eth0"
+				serverIPAddr   = "192.0.2.102"
+				serverCIDR     = serverIPAddr + "/24"
+				clientCIDR     = "192.0.2.101/24"
+			)
+			nodeList := libnode.GetAllSchedulableNodes(kubevirt.Client())
+			Expect(nodeList.Items).NotTo(BeEmpty(), "schedulable kubernetes nodes must be present")
+			nodeName := nodeList.Items[0].Name
+
+			const (
+				linuxBridgeConfNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"mynet\", \"plugins\": [{\"type\": \"bridge\", \"bridge\": \"%s\"}]}"}}`
+				linuxBridgeNADName = "bridge0"
+			)
+			bridgeNAD := fmt.Sprintf(linuxBridgeConfNAD, linuxBridgeNADName, testsuite.GetTestNamespace(nil), "br10")
+			namespace := testsuite.GetTestNamespace(nil)
+			Expect(libnet.CreateNetworkAttachmentDefinition(linuxBridgeNADName, namespace, bridgeNAD)).To(Succeed())
+
+			primaryIface := libvmi.InterfaceWithBindingPlugin(
+				"mynet1", v1.PluginBinding{Name: bindingName},
+			)
```

Comment:
I find it a bit misleading that the interface is called `primaryIface` but it actually connects to a secondary network.

Reply:
Right. Moved to use the primary with the Multus default flag.
Done